File: app/src/main/java/com/iita/akilimo/entities/CurrentFieldYield.java
package com.iita.akilimo.entities;

import io.objectbox.annotation.Entity;
import io.objectbox.annotation.Id;
import io.objectbox.annotation.Transient;
import lombok.Data;

@Data
@Entity
public class CurrentFieldYield {

    @Id
    long id;
    private double yieldAmount;
    private String fieldYieldLabel;

    @Transient
    private int imageId;
}

File: app/src/main/java/com/iita/akilimo/entities/CurrentFieldYield.kt
package com.iita.akilimo.entities

import androidx.room.PrimaryKey
import com.j256.ormlite.field.DatabaseField
import com.j256.ormlite.table.DatabaseTable


@DatabaseTable(tableName = "current_field_yield")
open class CurrentFieldYield {

    @DatabaseField(columnName = "id", generatedId = true)
    var id: Int? = null
    var yieldAmount = 0.0
    var fieldYieldLabel: String? = null

    @Transient
    var imageId = 0


}